src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -53,15 +53,13 @@ class InputOption
     private $description;
 
     /**
-     * @param string                    $name        The option name
-     * @param string|array|null         $shortcut    The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
-     * @param int|null                  $mode        The option mode: One of the VALUE_* constants
-     * @param string                    $description A description text
-     * @param string|string[]|bool|null $default     The default value (must be null for self::VALUE_NONE)
+     * @param $shortcut The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
+     * @param $mode     The option mode: One of the VALUE_* constants
```

Comment:
IIUC we dont duplicate the type as per CS?

Reply:
oh wait, this is totally fine :+1: 

> When provided it MAY contain a Type to indicate what is expected
> https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/param.html#param

for now :facepalm: 

>  When provided it MUST contain a "Type" to indicate what is expected
> https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#59-param

Reply:
We settled on: PSR19 is not a thing and is outdated anyway, especially considering PHP8 + phpdoc.org is the most up-to-date specification/authority for annotations.